modell/src/test/kotlin/no/nav/tilbakekreving/tilstand/AvventerKravgrunnlagHendelseTest.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.tilbakekreving.tilstand
+
+import io.kotest.inspectors.forOne
+import io.kotest.matchers.equals.shouldBeEqual
+import io.kotest.matchers.shouldBe
+import no.nav.tilbakekreving.Tilbakekreving
+import no.nav.tilbakekreving.behov.BehovObservatørOppsamler
+import no.nav.tilbakekreving.behov.FagsysteminfoBehov
+import no.nav.tilbakekreving.hendelse.KravgrunnlagHendelse
+import no.nav.tilbakekreving.kontrakter.ytelse.Fagsystem
+import no.nav.tilbakekreving.kontrakter.ytelse.Ytelsestype
+import no.nav.tilbakekreving.opprettTilbakekrevingEvent
+import org.junit.jupiter.api.Test
+
+class AvventerKravgrunnlagHendelseTest {
+    @Test
+    fun `tilbakekreving i AvventerKravgrunnlag går videre med Kravgrunnlag`() {
```

Comment:
Pirk: har casing noe å si for kotlin-tester?
```suggestion
    fun `Tilbakekreving i AvventerKravgrunnlag går videre med Kravgrunnlag`() {
```

Reply:
Nei, men er bare vant med at man kun bruker store bokstaver dersom det refererer til kodegreier